keyboards/viktus/osav2_numpad/info.json

```diff
@@ -0,0 +1,245 @@
+{
+    "manufacturer": "Viktus Design LLC",
+    "keyboard_name": "OSAv2 Numpad",
+    "maintainer": "BlindAssassin111",
+    "url": "https://viktus.design",
+    "usb": {
+        "device_version": "1.0.0",
+        "vid": "0x5644",
+        "pid": "0x4F4E"
+    },
+    "bootloader": "atmel-dfu",
+    "processor": "atmega32u4",
+    "features": {
+        "rgblight": true,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": false,
+        "mousekey": false,
```

Comment:
Is there a reason for these two to be disabled?

Reply:
Was from when I was removing everything from rules.mk. I had this in two other info.JSON's for boards that merged recently and it was never brought up, so didn't notice.

Reply:
Generally, we like having extra keys and mousekeys enabled, as they're commonly used. And the QMK Configurator doesn't have a way to toggle the features, so it uses whatever these are defaulted to. (eg, it's easier to have enabled)